### DISCUSSION TITLE
Two column layout with explanation and code

### DISCUSSION THREAD

**Discussion by entuziaz** [shuding/nextra#2896] [Q&A]
Is it possible to achieve a two-column REST API reference documentation on Nextra with explanation on the left and code snippet on the right?
As shown in the attached image.

<img width="1175" alt="image" src="https://github.com/shuding/nextra/assets/28672755/2fb631ba-16b2-4f10-bcdb-610e27d8dff6">

@nathanhammond @fanweixiao @benoror @benjie @styfle

**Comment by benoror** [comment]
It's just a matter of laying out your components: Checkout this example from @baseql for inspiration:
- https://docs.baseql.com/usage/querying
- https://github.com/BaseQL/baseql-docs

**Reply by entuziaz** [reply]
Oh, thanks for the pointer @benoror . I'll try reworking the layout and give a feedback.
